test/modules/nfts/createNft.test.ts

```diff
@@ -3,207 +3,215 @@ import spok, { Specifications } from 'spok';
 import { Keypair } from '@solana/web3.js';
 import { UseMethod } from '@metaplex-foundation/mpl-token-metadata';
 import { JsonMetadata, MetaplexFile, Nft } from '@/index';
-import { metaplex, spokSamePubkey, spokSameBignum } from '../../helpers';
+import { metaplex, spokSamePubkey, spokSameBignum, killStuckProcess } from '../../helpers';
+
+killStuckProcess();
 
 test('it can create an NFT with minimum configuration', async (t: Test) => {
-	// Given we have a Metaplex instance.
-	const mx = await metaplex();
-
-	// And we uploaded an image.
-	const imageFile = new MetaplexFile('some_image', 'some-image.jpg');
-	const imageUri = await mx.storage().upload(imageFile);
-
-	// And we uploaded some metadata containing this image.
-	const metadataUri = await mx.storage().uploadJson<JsonMetadata>({
-		name: 'JSON NFT name',
-		description: 'JSON NFT description',
-		image: imageUri,
-	});
-
-	// When we create a new NFT with minimum configuration.
-	const { nft } = await mx.nfts().createNft({
-		name: 'On-chain NFT name',
-		uri: metadataUri,
-	});
-
-	// Then we created and retrieved the new NFT and it has appropriate defaults.
-	spok(t, nft, {
-		$topic: 'nft',
-		name: 'On-chain NFT name',
-		uri: metadataUri,
-		metadata: {
-			name: 'JSON NFT name',
-			description: 'JSON NFT description',
-			image: imageUri,
-		},
-		sellerFeeBasisPoints: 500,
-		primarySaleHappened: false,
-		updateAuthority: spokSamePubkey(mx.identity().publicKey),
-		creators: [
-			{
-				address: spokSamePubkey(mx.identity().publicKey),
-				share: 100,
-				verified: true,
-			},
-		],
-		collection: null,
-		uses: null,
-	} as unknown as Specifications<Nft>);
+  // Given we have a Metaplex instance.
```

Comment:
I'm realizing that this entire file got reformatted, but this is expected since `.prettierrc` has ` "tabWidth": 2` this is bound to happen.
I'm just wondering why it was still 4-tab spaced .. maybe we should include `./test` for that `lint:fix` task?

Reply:
Agreed. Could you please add the test folder to the lint script on this PR? 🙂